src/main/java/figures/Circle.java

```diff
@@ -0,0 +1,23 @@
+package figures;
+
+import behavior.Figure;
+
+public class Circle extends Figure {
+    private final double radius;
+    private String name = "Circle";
```

Comment:
if you want to save the name of the figure let's pass it through the constructor or we can remove it